test/libraries/TickBitmap.t.sol

```diff
@@ -167,8 +167,8 @@ contract TickBitmapTest is Test, GasSnapshot {
     }
 
     function test_nextInitializedTickWithinOneWord_lteFalse_onBoundary_gas() public {
-        bitmap.nextInitializedTickWithinOneWord(255, 1, false);
         snapStart("nextInitializedTickWithinOneWord_lteFalse_onBoundary");
+        bitmap.nextInitializedTickWithinOneWord(255, 1, false);
```

Comment:
i see all the tests run with a tickSpacing of 1... would you be able to add a test with more of a range of tick vs tickSpacing? maybe a fuzz test would be ideal

Reply:
I'm afraid not.

Reply:
Added a fuzz test.